tensorflow/compiler/mlir/tensorflow/tests/BUILD

```diff
@@ -5,6 +5,10 @@ package(licenses = ["notice"])
 glob_lit_tests(
     data = [":test_utilities"],
     driver = "@llvm-project//mlir:run_lit.sh",
+    tags_override = {
```

Comment:
Do you perhaps have issues you could attach to these as to why they are disabled for rocm?

Reply:
We have not yet root-cause analysis of why these tests are failing on the rocm side. They showed up as regressions, when we did a (upstream repo ---> rocm fork) sync last week.  They were tagged as no_rocm at that point.

There are two processes going on concurrently on the ROCm side. 
* One is to get the contents of our fork in sync with the upstream repo. This PR is a part of that process
* At the same time, we are also looking into the existing test failures on the ROCm platform, and fixing them. These failures will be looked at as part of that process. I will post more information once I have it.